### PR TITLE
Tutorial/discord bot

### PR DESCRIPTION
**Pull request by Aggis15**
Making this as a draft as it's not ready to be merged yet. @evoxmusic any chance you can take a look as to why my page isn't loading?

### CODE REVIEW

File: .meta/.schemas/guides.json [hunk 1]
```diff
@@ -52,7 +52,7 @@
...
+        "enum": ["type: guide","type: post","type: tutorial","type: engineering","domain: buffers","domain: cli","domain: config","domain: dependencies","domain: networking","domain: observability","domain: operations","domain: platforms","domain: security","domain: sinks","domain: sources","domain: tests","domain: topology","domain: transforms","domain: ux","domain: website","domain: services","language: php","language: kotlin","language: java","language: javascript","language: python","language: rust","language: go","language: ruby","database: mysql","database: postgresql","database: mongodb","framework: hasura","framework: laravel","framework: springboot","framework: nodejs","framework: flask","framework: jhipster","framework: gin","framework: rails","framework: django","framework: deno","framework: strapi","framework: nuxtjs","framework: discordpy","technology: kubernetes","technology: docker","technology: kotlin","technology: qovery","cloud_provider: aws","cloud_provider: digital_ocean","cloud_provider: scaleway"]
```
Comment (evoxmusic) [inline .meta/.schemas/guides.json:55]: did you modified this file manually or it was modified automatically?

File: scripts/Gemfile.lock [hunk 2]
```diff
@@ -58,4 +59,4 @@ RUBY VERSION
    ruby 3.0.1p64
 
 BUNDLED WITH
-   2.2.20
+   2.2.24
```
Comment (evoxmusic) [inline scripts/Gemfile.lock:62]: why did you update it?

File: scripts/Gemfile.lock [hunk 3]
```diff
@@ -39,6 +39,7 @@ GEM
 
 PLATFORMS
   x86_64-darwin-20
+  x86_64-linux
```
Comment (evoxmusic) [inline scripts/Gemfile.lock:42]: really required? it was modified automatically?

File: .meta/.schemas/guides.json (hunk 1 above)
Comment (Aggis15) [reply .meta/.schemas/guides.json:55]: I updated this file manually so I can add the discordpy framework. Was that not required?

File: scripts/Gemfile.lock (hunk 2 above)
Comment (Aggis15) [reply scripts/Gemfile.lock:62]: This was updated automatically. Not entirely sure how or when. Should I downgrade ruby?

File: scripts/Gemfile.lock (hunk 3 above)
Comment (Aggis15) [reply scripts/Gemfile.lock:42]: This was also modified automatically. Should I remove that line?

File: .meta/.schemas/guides.json (hunk 1 above)
Comment (evoxmusic) [reply .meta/.schemas/guides.json:55]: `frameworks.toml` is ok to be modified 👍🏽 but the `guides.json` must not be updated manually.

File: scripts/Gemfile.lock (hunk 2 above)
Comment (evoxmusic) [reply scripts/Gemfile.lock:62]: no prob

File: scripts/Gemfile.lock (hunk 3 above)
Comment (evoxmusic) [reply scripts/Gemfile.lock:42]: No, it is good

Comment (Aggis15) [reply scripts/Gemfile.lock:42]: That's awesome. Any reason that the guide isn't loading now? :')

**Comment by evoxmusic** [comment]
@Aggis15 you need to add this into the `team.toml` (same folder as `frameworks.toml`)

```toml
[[team]]
id = "Aggis15"
avatar = "https://github.com/Aggis15.png"
github = "https://github.com/Aggis15"
keybase = "https://keybase.io/Aggis15"
name = "Angelos Rinas"
bio = """\
I am a student from Greece, currently learning Python, but planning on moving to other languages when I feel like I'm competent enough with Python.
\"""
```

**Comment by Aggis15** [comment]
Yuuuup, that sure did it. Thank you so much!

**Comment by Aggis15** [comment]
Tutorial is ready. Please let me know if I should do anything else on my end.

metadata.js was edited automatically once you told me to add that line in the team.toml folder but I haven't commited it here. Let me know if I should do that and I'll do it ASAP.

**Comment by evoxmusic** [comment]
yes you have to commit it :)

**Comment by Aggis15** [comment]
Commited :)

**Pull request merged**